### PR TITLE
PLAT-331: run unit tests on windows

### PR DESCRIPTION
**Pull request by afiskon**
Run unit tests on Windows.

### CODE REVIEW

File: .github/workflows/all-checks.yaml [hunk 1]
```diff
@@ -75,3 +75,46 @@ jobs:
         with:
           name: coverage-out
           path: ${{env.GOPATH}}/src/github.com/insolar/assured-ledger/ledger-core/v2/coverage.out
+  tests-on-windows:
+    name: windows-checks
+    runs-on: windows-latest
+    env: # we have to specify fake BUILD_DATE and BUILD_TIME here because git-date-time.sh doesn't work on Windows
+      GOPATH: D:/a/assured-ledger/assured-ledger/go
+      WORKDIR: D:/a/assured-ledger/assured-ledger/go/src/github.com/insolar/assured-ledger/ledger-core/v2
+      BUILD_DATE: 20200101
```
Comment (cyraxred) [inline .github/workflows/all-checks.yaml:84]: "time /T" and "date /T" do not work as well?

File: ledger-core/v2/insolar/defaults/defaults_test.go [hunk 2]
```diff
@@ -55,6 +56,10 @@ var cases = []tCase{
 }
 
 func TestDefaults(t *testing.T) {
+	if runtime.GOOS == "windows" {
```
Comment (cyraxred) [inline ledger-core/v2/insolar/defaults/defaults_test.go:59]: Please use filepath package to fix the test, not to skip it

File: ledger-core/v2/network/hostnetwork/hostnetwork_test.go [hunk 3]
```diff
@@ -170,6 +171,10 @@ func (s *hostSuite) Stop() {
 }
 
 func TestNewHostNetwork(t *testing.T) {
+	if runtime.GOOS == "windows" { // TODO FIXME
```
Comment (cyraxred) [inline ledger-core/v2/network/hostnetwork/hostnetwork_test.go:174]: Please add and mention here a JIRA taks

File: ledger-core/v2/virtual/integration/method_api_test.go [hunk 4]
```diff
@@ -108,6 +109,10 @@ func TestVirtual_Method_API(t *testing.T) {
 
 // 10 parallel executions
 func TestVirtual_Scenario1(t *testing.T) {
+	if runtime.GOOS == "windows" { // TODO FIXME
```
Comment (cyraxred) [inline ledger-core/v2/virtual/integration/method_api_test.go:112]: Please add and mention here a JIRA taks

File: ledger-core/v2/virtual/integration/method_test.go [hunk 5]
```diff
@@ -106,6 +107,10 @@ func Method_PrepareObject(ctx context.Context, server *utils.Server, prototype r
 }
 
 func TestVirtual_Method_WithoutExecutor(t *testing.T) {
+	if runtime.GOOS == "windows" { // TODO FIXME
```
Comment (cyraxred) [inline ledger-core/v2/virtual/integration/method_test.go:110]: Please add and mention here a JIRA taks

Comment (afiskon) [reply ledger-core/v2/virtual/integration/method_test.go:110]: +

File: ledger-core/v2/virtual/integration/method_api_test.go (hunk 4 above)
Comment (afiskon) [reply ledger-core/v2/virtual/integration/method_api_test.go:112]: done

File: ledger-core/v2/network/hostnetwork/hostnetwork_test.go (hunk 3 above)
Comment (afiskon) [reply ledger-core/v2/network/hostnetwork/hostnetwork_test.go:174]: done

File: ledger-core/v2/insolar/defaults/defaults_test.go (hunk 2 above)
Comment (afiskon) [reply ledger-core/v2/insolar/defaults/defaults_test.go:59]: done

File: .github/workflows/all-checks.yaml (hunk 1 above)
Comment (afiskon) [reply .github/workflows/all-checks.yaml:84]: I don't know. As you might guess there are not many Windows users among us.

**Pull request merged**